linebot/models/background.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+
+#  Licensed under the Apache License, Version 2.0 (the "License"); you may
+#  not use this file except in compliance with the License. You may obtain
+#  a copy of the License at
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations
+#  under the License.
+
+"""linebot.models.emojis module."""
+
+from __future__ import unicode_literals
+
+from abc import ABCMeta
+
+from future.utils import with_metaclass
+
+from .base import Base
+
+
+class Background(with_metaclass(ABCMeta, Base)):
+    """Background."""
+
+    def __init__(self, **kwargs):
+        """__init__ method.
+
+        :param kwargs:
+        """
+        super(Background, self).__init__(**kwargs)
+
+        self.type = None
+
+
+class LinearGradientBackground(Background):
+    """LinearGradientBackground."""
+
+    def __init__(self, angle=None, start_color=None, end_color=None,
```

Comment:
[must]
`angle`, `start_color` and `end_color` are required when background.type is linearGradient.